.github/workflows/warm-up-vercel.yml

```diff
@@ -12,8 +12,7 @@ on:
 jobs:
   fetch:
     name: "Fetch"
-    if: |
-      !github.event.deployment_status || github.event.deployment_status.state == 'success'
+    if: github.event_name == 'schedule' || github.event.deployment_status.state == 'success'
```

Comment:
The logic should be the same here. I had to use `|` previously because `if: !` was a syntax error.